zerver/openapi/zulip.yaml

```diff
@@ -790,6 +790,71 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/NonExistingStreamError'
+  /users/me/subscriptions/muted_topics:
+    patch:
+      description: Toggle muting for a specific topic the user is subscribed
+        to.
+      parameters:
+      - name: stream
+        in: query_function
+        description: The name of the stream where the topic is located.
+        schema:
+          type: string
+        example: Verona
+        required: true
+      - name: topic
+        in: query
+        description: The name of the topic to (un)mute. Please note that this
+          does not have to exist in the stream in order for the query to
+          succeed.
+        schema:
+          type: string
+        example: dinner
+        required: true
+      - name: op
+        in: query
+        description: Whether to `add` or `remove` the topic from the list of
+          topics muted by the user.
```

Comment:
Do we need to talk about a "list of topics"?  I don't think so.  We should instead talk about the fact that it's within a stream.